generate-release-vars.sh

```diff
@@ -67,4 +72,5 @@ source_ami_al2       = "$ami_name_al2_x86"
 source_ami_al2arm    = "$ami_name_al2_arm"
 source_ami_al2022    = "$ami_name_al2022_x86"
 source_ami_al2022arm = "$ami_name_al2022_arm"
+repo_version_al2022  = "$repo_version_al2022"
```

Comment:
nit: calling this `repo_version` seems a little vague to me here as yum repos usually refer to the places where individual packages are installed from, dnf refers to this version in particular as the "distribution release", maybe we can rename to `distribution_version_al2022` or `distribution_release_al2022`?

from https://linuxcommandlibrary.com/man/dnf

```
--releasever=<release>
    Configure DNF as if the distribution release was <release>. This can affect cache paths, values in configuration files and mirrorlist URLs.
```

Reply:
Will update it! Thanks!